views/Legend/ChoroplethLegend.svelte

```diff
@@ -294,12 +305,18 @@
     justify-content: space-between;
   }
 
```

Comment:
<img width="314" alt="image" src="https://user-images.githubusercontent.com/17497583/170079354-38087422-9a0e-452b-a000-48d79c257f67.png">

Ich habe ein extremes Beispiel gemacht mit einer viel zu langen Masseinheit. Aber das Problem ist das selbe wenn anfangs und Endzahl viel zu lang sind. 
Soll keine Kritik an dich oder den Code sein, mehr dass wir in diesem Fall vielleicht doch auch noch einen Warnhinweis über grosse Zahlen ausspielen könnten. Eine andere Lösung ist das kürzen der verwendeten Zahlen zu einer anderen Einheit (kg zu t oder 10'000 als 10k darstellen).
Und sonst finde ich muss sich der Kartenersteller dem einfach bewusst sein.